py/autoevals/test_llm.py

```diff
@@ -136,7 +136,7 @@ def test_battle():
         )
 
         print(response.as_json(indent=2))
-        assert response.score == (1 if use_cot else 0)
+        assert response.score == 1
```

Comment:
This was a hack -- seems like even _without_ CoT, gpt-4-1106-preview gets the right answer :)